apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/CarrierClaimTransformerTest.java

```diff
@@ -68,6 +68,37 @@ public void transformSampleARecord() throws FHIRException {
     assertMatches(claim, eobWithoutLastUpdated, Optional.of(true));
   }
 
+  /**
+   * Verifies that {@link
+   * gov.cms.bfd.server.war.stu3.providers.CarrierClaimTransformer#transform(Object)} works as
+   * expected when run against the {@link StaticRifResource#SAMPLE_A_CARRIER} {@link CarrierClaim}.
+   *
```

Comment:
Missing context about why and what "as expected" means; it will be helpful to have some context around the issue for posterity

Reply:
done